python/paddle/fluid/tests/unittests/test_nearest_interp_v2_op.py

```diff
@@ -22,7 +22,7 @@
 import paddle.nn as nn
 import paddle
 from paddle.nn.functional import interpolate
-from paddle._C_ops import final_state_nearest_interp
+from paddle import _C_ops
```

Comment:
这里是因为原来 NPU 下单测 `test_nearest_interp_v2_op_npu` 在从 `test_nearest_interp_v2_op` import numpy 的参考实现（`nearest_neighbor_interp_np`）时候会报错，报错如下：

```text
2022-08-17 00:32:35 1/1 Test #93: test_nearest_interp_v2_op_npu ....***Failed    3.95 sec
2022-08-17 00:32:35 W0817 00:32:32.922722   813 init.cc:288] AVX is available, Please re-compile on local machine
2022-08-17 00:32:35 Hint: Your machine support AVX, but the installed paddlepaddle doesn't have avx core. Hence, no-avx core with worse preformance will be imported.
2022-08-17 00:32:35 If you like, you could reinstall paddlepaddle by 'python -m pip install --force-reinstall paddlepaddle-gpu[==version]' to get better performance.
2022-08-17 00:32:35 The original error is: cannot import name 'core_avx' from 'paddle.fluid' (/workspace/npu_dev/Paddle/build/python/paddle/fluid/__init__.py)
2022-08-17 00:32:35 /workspace/npu_dev/Paddle/build/python/paddle/fluid/framework.py:189: UserWarning: We will fallback into legacy dygraph on NPU/XPU/MLU/IPU/ROCM devices. Because we only support new eager dygraph mode on CPU/GPU currently. 
2022-08-17 00:32:35   "We will fallback into legacy dygraph on NPU/XPU/MLU/IPU/ROCM devices. Because we only support new eager dygraph mode on CPU/GPU currently. "
2022-08-17 00:32:35 Traceback (most recent call last):
2022-08-17 00:32:35   File "/workspace/npu_dev/Paddle/tools/test_runner.py", line 69, in <module>
2022-08-17 00:32:35     main()
2022-08-17 00:32:35   File "/workspace/npu_dev/Paddle/tools/test_runner.py", line 51, in main
2022-08-17 00:32:35     module = importlib.import_module(module_name)
2022-08-17 00:32:35   File "/opt/conda/lib/python3.7/importlib/__init__.py", line 127, in import_module
2022-08-17 00:32:35     return _bootstrap._gcd_import(name[level:], package, level)
2022-08-17 00:32:35   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
2022-08-17 00:32:35   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
2022-08-17 00:32:35   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
2022-08-17 00:32:35   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
2022-08-17 00:32:35   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
2022-08-17 00:32:35   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
2022-08-17 00:32:35   File "/workspace/npu_dev/Paddle/build/python/paddle/fluid/tests/unittests/npu/test_nearest_interp_v2_op_npu.py", line 29, in <module>
2022-08-17 00:32:35     from test_nearest_interp_v2_op import nearest_neighbor_interp_np
2022-08-17 00:32:35   File "../test_nearest_interp_v2_op.py", line 25, in <module>
2022-08-17 00:32:35     from paddle._C_ops import final_state_nearest_interp
2022-08-17 00:32:35 ImportError: cannot import name 'final_state_nearest_interp' from 'paddle.fluid.core_noavx.ops' (/workspace/npu_dev/Paddle/build/python/paddle/_C_ops.py)
2022-08-17 00:32:35 Errors while running CTest
2022-08-17 00:32:35 0% tests passed, 1 tests failed out of 1
2022-08-17 00:32:35 Total Test time (real) =   3.96 sec
2022-08-17 00:32:35 The following tests FAILED:
2022-08-17 00:32:35 	 93 - test_nearest_interp_v2_op_npu (Failed)
```

看起来是刚刚 merge 的 #45148 新增的这一行直接从 _C_ops import 了 `final_state_nearest_interp`，但在 NPU 下貌似没有这个算子。为了避免这个问题，修改为该文件全局仅 import 到 _C_ops，修改后 NPU 测试全部通过。